src/deepsparse/transformers/helpers.py

```diff
@@ -109,6 +110,11 @@ def get_onnx_path_and_configs(
         tokenizer_path = _get_file_parent(
             zoo_model.deployment.default.get_file(_MODEL_DIR_TOKENIZER_NAME).path
         )
+        tokenizer_config_path = zoo_model.deployment.default.get_file(
+            _MODEL_DIR_TOKENIZER_CONFIG_NAME
+        )
+        if tokenizer_config_path is not None:
+            tokenizer_config_path.path  # trigger download of tokenizer_config
```

Comment:
```suggestion
            tokenizer_config_path.download()
```

Reply:
don't want to force download, going through `.path` route to use cache, can update comment if confusing

Reply:
Calling path also downloads the needed artifacts right?

Reply:
The comment is clear, it just reads weird because the property access is being treated as a method. It makes sense because we all know that path is actually a method, but for those who don't know I imagine its confusing

Reply:
right, that's why the comment was added, since it may be unclear otherwise that accessing `path` may trigger a download

Reply:
Does `.download()` not use the `cache`? 

For the scope of this PR it seems okay, but maybe using `.path` is adding more confusion than making the code clearer. For example just looking at `Model(...).path` it is not very clear whether this will download the model(we know it will download the model ofc) or just `return` the local path where model is/will be stored.